chore: [sc-19514] Rewrite Ensign for Data Scientists docs page (#615)
rotationalio/ensign: docs/content/examples/data_scientists.md

We'll write this as a function called `handle`. The first step is to unmarshal each new document from [MessagePack](https://msgpack.org/index.html) format into json (the Baleen application publishes documents in msgpack because it's more efficient!):

```python
    aasync def handle_event(self,event):
        """
        Decode and ack the event.
        ----------------
        Unpacking of the event message and working on the article content for
        NLP Magic
        """

        try:
            print(type(event.data))
            data = msgpack.unpackb(event.data)

>         except json.JSONDecodeError:
>             print("Received invalid JSON in event payload:", event.data)

Reviewer: ```suggestion
        except Exception:
            print("Received invalid msgpack data in event payload:", event.data)
```

We're using msgpack to decode this so no JSON error is raised here.

We'll use the amazing [BeautifulSoup](https://www.crummy.com/software/BeautifulSoup/bs4/doc/) library:

```python

>     async def handle_event(self,event):

Reviewer: ```suggestion
    async def parse_event(self, event):
```

        """

>         Decode and ack the event.
>         ----------------
>         Unpacking of the event message and working on the article content for
>         NLP Magic

Reviewer: ```suggestion
        Decode the msgpack payload, in preparation for applying our NLP "magic"
```
